api/scripts/run-docker.sh

```diff
@@ -59,6 +66,9 @@ if [ "$1" == "migrate" ]; then
 elif [ "$1" == "serve" ]; then
     if [ $# -eq 2 ]; then go_to_sleep "$2"; fi
     serve
+elif [ "$1" == "run-task-processor" ]; then
+    if [ $# -eq 2 ]; then go_to_sleep "$2"; fi
```

Comment:
# nit: should we get rid of  `go_to_sleep` here? @matthewelwell  what do you think?

Reply:
> Yeah, you're right, we definitely _should_, but I don't think it's necessary here. It doesn't look like that functionality is used anywhere but it's technically part of the public API of our docker image and I don't want to have to major version our docker image(s).

But we can choose to not add that here, right? Since that's not part of the public API yet?

Reply:
> > Yeah, you're right, we definitely _should_, but I don't think it's necessary here. It doesn't look like that functionality is used anywhere but it's technically part of the public API of our docker image and I don't want to have to major version our docker image(s).
> 
> But we can choose to not add that here, right? Since that's not part of the public API yet?

Sorry, that's true, yes!

Reply:
Done.